.github/workflows/main.yml

```diff
@@ -78,6 +80,8 @@ jobs:
         pip install -r dev_requirements.txt
         pip install WxPython==4.1.1
         pip install -e .
+        pip uninstall kenlm
+        pip install kenlm --global-option="--max_order=12"

```

Comment:
Is there a version we could/should pin?

Reply:
I had it in the language module README, but I've added it to the general README as well.